### ISSUE TITLE
Using this API to interface with Blue Iris or Shinobi ?

### ISSUE THREAD

**Issue by gszakacs** [kroo/wyzecam#25]
Hi, 
I am wondering if this would be a better solution for streaming wyze v3 video/audio to  blue iris, instead of using the NFS wyzehack and js script solution.
If using your API, t is possible to feed the stream to BI, could you provide an example ? This API could be extremely popular if there was an example provided to interface the video stream to BI, shinobi, etc.
Thanks

**Comment by kroo** [comment]
hi @gszakacs, yes, it's definitely possible, and was actually the main reason I put this together.  I have an RTSP bridge app (exposing all the local cameras on your network as RTSP streams) working as a prototype, but it's not quite ready for a public release.   If you're interested in testing it out and providing feedback, I'm happy to share -- alternatively I'm hoping to clean it up and publish it once I have the kinks worked out (mostly performance issues and frequent disconnects with larger numbers of cameras, right now...).

**Comment by gszakacs** [comment]
Happy to test ! I only have 3 wyze cams (2 x v3 and 1 x v2) + a yi home, a foscam c1 and a few ESP32 cams. I am also wondering if you could implement streaming easier based on the idea described here: https://github.com/n1ckyrush/wyzecam-hls.

**Comment by noelhibbard** [comment]
> hi @gszakacs, yes, it's definitely possible, and was actually the main reason I put this together. I have an RTSP bridge server (exposing all the local cameras on your network as RTSP streams) working as a prototype, but it's not quite ready for a public release. If you're interested in testing it out and providing feedback, I'm happy to share -- alternatively I'm hoping to clean it up and publish it once I have the kinks worked out (mostly performance issues and frequent disconnects with larger numbers of cameras, right now...).

Man you would be a hero to a lot of people if you pulled this off!

**Comment by calvinnelms** [comment]
@kroo I would be happy to test.  I have 2 wyze cams (along with a few non wyze cams) and trying to import that video feed into Synology Surveillance Station.

**Comment by noelhibbard** [comment]
I was able to spin up an nginx server with the RTMP module and then modified the streaming_video.py sample to pipe the frames to ffmpeg which then sends the stream to my nginx server. It's working well and only has about 4 secs lag when playing back on my iPhone with VLC.

The ffmpeg command looks something like this:
```ffmpeg -f h264 -i - -vcodec copy -f flv rtmp://<yourserverip>:1935/live/doorbell```

The wife is hovering because I spent the past hour throwing this together. When it's safe I will post more detail and a code sample.

**Comment by DennisGarvey** [comment]
> hi @gszakacs, yes, it's definitely possible, and was actually the main reason I put this together. I have an RTSP bridge server (exposing all the local cameras on your network as RTSP streams) working as a prototype, but it's not quite ready for a public release. If you're interested in testing it out and providing feedback, I'm happy to share -- alternatively I'm hoping to clean it up and publish it once I have the kinks worked out (mostly performance issues and frequent disconnects with larger numbers of cameras, right now...).

Would you mind sharing? I have 3 v3 cams

**Comment by mistersidd23** [comment]
> > hi @gszakacs, yes, it's definitely possible, and was actually the main reason I put this together. I have an RTSP bridge server (exposing all the local cameras on your network as RTSP streams) working as a prototype, but it's not quite ready for a public release. 

Yeah - I'd love to test this as well.  running 3 v2s, (and a usb webcam in blue iris to achieve limited audio) - would like to get my v3 and esp32 into BI as well.

**Comment by kroo** [comment]
For those looking to beta test, I just put up a very early version of what I've been working on here: https://github.com/kroo/wyze-rtsp-bridge

**Comment by mistersidd23** [comment]
Thank you Elliot!
